editor/src/clj/editor/model_scene.clj

```diff
@@ -236,12 +236,11 @@
 
 (defn- render-outline [^GL2 gl render-args renderables rcount]
   (assert (= 1 rcount) "Batching is disabled in the editor for simplicity.")
-  (let [pass (:pass render-args)
-        renderable (first renderables)
-        node-id (:node-id renderable)
-        request-id [::outline node-id]]
-    (assert (= pass/outline pass))
-    (render/render-aabb-outline gl render-args request-id renderables rcount)))
+  (when (= pass/outline (:pass render-args))
+    (let [renderable (first renderables)
+          node-id (:node-id renderable)
+          request-id [::outline node-id]]
+      (render/render-aabb-outline gl render-args request-id renderables rcount))))
```

Comment:
Mostly indentation. Wrapped in `(when (= pass/outline (:pass render-args)) ...)` instead of asserting on the pass. We needed to add the renderable to `pass/selection` or `pass/opaque-selection` for it to be considered selectable.